src/cli.ts

```diff
@@ -5,7 +5,24 @@
 import * as meow from 'meow';
 import {CommandRunner} from './commands';
 import {Global} from './Global';
+import updateNotifier = require('update-notifier');
```

Comment:
Why are you using a require statement here? 🤔 In TypeScript you should use regular imports wherever possible...